docs/en/latest/admin-api.md

```diff
@@ -94,11 +94,10 @@ Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/ap
 | create_time      | False                                    | Auxiliary   | epoch timestamp in second, will be created automatically if missing                                                                                                                                                                                                                                                                                                                                                                | 1602883670                                           |
 | update_time      | False                                    | Auxiliary   | epoch timestamp in second, will be created automatically if missing                                                                                                                                                                                                                                                                                                                                                                | 1602883670                                           |
 
-There are three points that need special attention:
+There are two points that need special attention:
 
-- In addition to the required `uri`/`uris`, you must select at least one of the four categories `plugin`, `script`, `upstream`/`upstream_id` and `service_id`.
 - For the same type of parameters, such as `host` and `hosts`, `remote_addr` and `remote_addrs` cannot exist at the same time, only one of them can be selected. If enabled at the same time, the API will respond with an error.
-- When using `vars`, obtaining the value of `cookie` is case-sensitive. For example: `var` is "cookie\_`x_foo`" and `var` is "cookie\_`X_Foo`" means to get different `cookie`.
+- In `vars`, when getting the cookie value, the cookie name is distinguished by uppercase and lowercase letters. For example: `var` equals "cookie_x_foo" and `var` equals "cookie_X_Foo" means different `cookie`.
```

Comment:
> the cookie name is distinguished by uppercase and lowercase letters

We have a better term: "case-sensitive".

Reply:
Updated.